test/unit/alphabet/quality_test.cpp

```diff
@@ -0,0 +1,184 @@
+// ============================================================================
```

Comment:
since you have the generic test now, you can remove the individual tests for phred42, phred63, phred_legacy68